mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/JPAProperty.java

```diff
@@ -38,7 +38,7 @@
     @Column(name = "PROPERTY_ID", nullable = true)
     // TODO The columnNames are not interpreted, see OPENJPA-223 to fix
     // MAILBOX-186
-    @Index(name = "INDEX_PROPERTY_MSG_ID", columnNames = { "MAILBOX_ID", "MAIL_UID" })
+    //@Index(name = "INDEX_PROPERTY_MSG_ID", columnNames = { "MAILBOX_ID", "MAIL_UID" })
```

Comment:
IMO we should remove completly these 3 comments line.

**git** history and JIRA are here to remind us about it.

Could you please delete these lines?

Reply:
Yes.
I wanted to delete but existing of TODO, made me uncertain about it.